### DISCUSSION TITLE
Please kindly ask Milvus what are the search optimization schemes and whether it can be optimized to complete the search within 100ms

### DISCUSSION THREAD

**Discussion by lijiawen-LJW** [milvus-io/milvus#29513] [Q&A and General discussion]
Please kindly ask Milvus what are the search optimization schemes and whether it can be optimized to complete the search within 100ms

**Comment by yhmo** [comment]
Not sure what you mean "optimization schemas". Collection schema is defined according to your business requirement.

Search latency mainly depends on: 
1. number of entities
2. dimension of vectors
3. index parameters
4. search parameters

In most cases, if the number of entities is not huge, dimension is not very high, with a proper index, search latency is easily controlled under 100ms.

**Reply by lijiawen-LJW** [reply]
Okay, thanks
